dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -173,4 +174,19 @@ def update(self, df_series):
 
         self._update_helper(df_series, profile)
 
-        return self
\ No newline at end of file
+        return self
+
+    @property
+    def gini_impurity(self):
+        """
+        Property for gini impurity
```

Comment:
lets put a reference to the formula or show the formula used here. Also need a space between the description and the return.